main/res/values/styles.xml

```diff
@@ -380,7 +380,7 @@
     <style name="cgeoProgressdialogTheme" parent="Theme.AppCompat.Dialog.Alert">
         <item name="android:textColorPrimary">?text_color</item>
         <item name="android:textColor">?text_color</item>
-        <item name="android:windowBackground">?background_color</item>
+        <item name="android:windowBackground">@drawable/border_straight</item>
```

Comment:
How do you debug such stuff typically? I was following the parent declarations of Theme.AppCompat.Dialog.Alert until I was at item name="android:windowBackground">@drawable/abc_dialog_material_background</item and didn't find any reasonable way to continue there...